src/traits.rs

```diff
@@ -102,6 +102,10 @@ pub trait IntoHandle {
     /// let owned_handle: OwnedHandle = f.into_handle();
     /// # Ok::<(), io::Error>(())
     /// ```
+    #[deprecated(
```

Comment:
Should a deprecation notice be attached to the traits as well?

Reply:
Good spot, yes, we can deprecate `IntoFd` and related traits.

I didn't mark `FromFd` and friends as deprecated yet, because they have the `from_into_fd` utility functions, which some users use and find handy. Fortunately, it's just a blanket impl for any type that implements `From<OwnedFd>`, so it's not something users need to implement themselves.